kyuubi-rest-client/src/main/java/org/apache/kyuubi/client/api/v1/dto/GetTablesRequest.java

```diff
@@ -0,0 +1,90 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.kyuubi.client.api.v1.dto;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Objects;
+
+public class GetTablesRequest {
+  private String catalogName;
+  private String schemaName;
+  private String tableName;
+  private List<String> tableTypes;
+
+  public GetTablesRequest() {}
+
+  public GetTablesRequest(
+      String catalogName, String schemaName, String tableName, List<String> tableTypes) {
```

Comment:
```
String catalogName,
String schemaName,
String tableName,
List<String> tableTypes
```

Reply:
I try to modify it, but `dev/reformat` will revert my change.

Reply:
see https://github.com/apache/incubator-kyuubi/pull/2668/commits/e2bf779cf2b08833b27d0c1f4833dc4da387a8b1

Reply:
oh.. let's leave it